docs/user-guide/configuring/ignoring-code.md

```diff
@@ -0,0 +1,182 @@
+# Ignoring Code
+
+
+* [`ignorePatterns` in config files](#ignorepatterns-in-config-files)
+* [`.eslintignore`](#eslintignore)
+* [Using an Alternate File](#using-an-alternate-file)
+* [Using eslintIgnore in package.json](#using-eslintignore-in-packagejson)
+* [Ignored File Warnings](#ignored-file-warnings)
+* [Disabling Inline Comments](#disabling-inline-comments)
+
+## `ignorePatterns` in config files
+
+You can tell ESLint to ignore specific files and directories by `ignorePatterns` in your config files. Each value of `ignorePatterns` is the same pattern as each line of `.eslintignore` in the next section.
```

Comment:
```suggestion
You can tell ESLint to ignore specific files and directories by `ignorePatterns` in your config files. Each value of `ignorePatterns` is a glob pattern similar to what you would use on the command line.
```

I don't think referring to `.eslintignore` helps here, so I'd suggest removing it.